subsys/bindesc/bindesc_skip.S

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
This file should probably be named `bindesc_skip_x86.S` or, even better, x86/bindesc_skip.S`